src/components/fields/ListOfMediaField.vue

```diff
@@ -87,7 +87,7 @@
 </template>
 
 <script>
-import DownloadOutlineIcon from 'vue-material-design-icons/DownloadOutline.vue'
+import DownloadOutlineIcon from 'vue-material-design-icons/TrayArrowDown.vue'
```

Comment:
```suggestion
import TrayArrowDownIcon from 'vue-material-design-icons/TrayArrowDown.vue'
```
And adjust the component list and the template.